forest/cli/src/cli/info_cmd.rs

```diff
@@ -0,0 +1,159 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::time::Instant;
+
+use clap::Subcommand;
+use colored::*;
+use forest_blocks::tipset_keys_json::TipsetKeysJson;
+use forest_rpc_client::{
+    chain_get_name, chain_get_tipset, chain_head, start_time, wallet_default_address,
+};
+use forest_shim::econ::TokenAmount;
+use time::OffsetDateTime;
+
+use super::Config;
+use crate::cli::{handle_rpc_err, wallet_cmd::format_balance_string};
+
+#[derive(Debug, Subcommand)]
+pub enum InfoCommand {
+    Show,
+}
+
+#[derive(Debug, strum_macros::Display)]
+enum SyncStatus {
+    Ok,
+    Slow,
+    Behind,
+}
+
+pub struct NodeStatusInfo {
+    /// timestamp of how far behind the node is with respect to syncing to head
+    behind: u64,
+    /// Chain health calculated as percentage: amount of blocks in last finality
+    /// very healthy amount of blocks in a finality (900 epochs * 5 blocks per
+    /// tipset)
```

Comment:
I struggle to parse this sentence. Do you mean it's the number of blocks in the last 900 epochs divided by 900*5?

Reply:
Ah I omitted the divide symbol, or maybe fmt did. Updated. 
taken from here: https://github.com/filecoin-project/lotus/blob/eed1a098c625cbbd29e490ea4f7e5286b8f74f76/cli/info.go#L65

Reply:
I think the comment in Lotus could be a lot better. Let's rewrite our comment to be more readable.